addons/acr/vehicles/CfgAntiAir.hpp

```diff
@@ -0,0 +1,48 @@
+// AA
+class I_LT_01_AA_F;
+class UKSF_ACR_Wiesel_AA: I_LT_01_AA_F {
+    side = 0;
+    faction="UKSF_ACR";
+    displayName="Wiesel AA";
+    // editorPreview = QPATHTOEF(common,data\previews\Inegal_LandRover_Unarmed.jpg);
+    hiddenSelectionsTextures[]={
+        "a3\armor_f_tank\lt_01\data\lt_01_main_olive_co.paa",
+        "a3\armor_f_tank\lt_01\data\lt_01_at_olive_co.paa",
+        "a3\armor_f\data\camonet_aaf_digi_green_co.paa",
+        "a3\armor_f\data\cage_olive_co.paa"
+    };
+    crew = "UKSF_ACR_Armour_Crewman";
+    typicalCargo[] = {
+        "UKSF_ACR_Armour_Crewman"
+    };
+
+    class TransportMagazines {};
```

Comment:
We can make a macro to do this in one line. See ‘addons/main/script_macros.hpp’ for where to do this. It’ll be similar to INVENTORY_EMPTY
One thing to note, is removing inventory items is garbage in arms, so it’s necessary to verify every custom vehicle has the inventory you expect